gms/factories/src/main/java/com/linkedin/gms/factory/common/Neo4jDriverFactory.java

```diff
@@ -27,19 +27,23 @@
   @Value("${NEO4J_MAX_CONNECTION_ACQUISITION_TIMEOUT_IN_SECONDS:60}")
   private Long neo4jMaxConnectionAcquisitionTimeout;
 
-  @Value("${NEO4j_MAX_CONNECTION_LIFETIME_IN_HOURS:1}")
+  @Value("${NEO4j_MAX_CONNECTION_LIFETIME_IN_SECONDS:3600}")
   private Long neo4jMaxConnectionLifetime;
```

Comment:
in order to not break anyone who may already be providing this value, should we accept both but default to using `NEO4j_MAX_CONNECTION_LIFETIME_IN_SECONDS` unless `NEO4j_MAX_CONNECTION_LIFETIME_IN_HOURS` is set elsewhere?

Reply:
I can look into it. That approach would be more customer oriented.

Reply:
I've done another attempt, and has now kept the old setting. It will take precedence of the new setting so that current users won't have any issues.